meta-leda-components/recipes-sdv/sdv-core/sdv-rauc-bundle.bb

```diff
@@ -31,6 +31,9 @@ RAUC_SLOT_rootfs = "sdv-image-minimal"
 # which cannot be used from within the guest. RAUC would fail on installing a "wic.qcow2" file.
 RAUC_SLOT_rootfs[fstype] = "ext4"
 
+# Set RAUC bundle version to git tag or git commit in case no tag is available
+RAUC_BUNDLE_VERSION = "${@bb.process.run('GIT_CEILING_DIRECTORIES= git describe --tags --always')[0]}"
```

Comment:
I'm unsure about calling git to determine the bundle version here, whether this is a good solution.
I hoped for a more sustainable or specific solution, something like "RAUC_BUNDLE_VERSION = "${IMAGE_ID}"
Temporarily, we can go with this though, as it technically works.